### PR TITLE
[W4.11e][W10-3]Adeel Dhedhi Bhyria

### PR DESCRIPTION
**Pull request by adeeldb**
Added a test to UtilsTest.java to test Utils#isAnyNull(Object…​) method.

### CODE REVIEW

**Comment by adeeldb** [comment]
Reverted and moved the work for LO W5.7 onto a fresh branch.

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -36,6 +36,32 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() throws Exception {
+        // No objects
+        assertNoNulls();
```
Comment (AceCentury) [inline test/java/seedu/addressbook/common/UtilsTest.java:42]: Nice work for spotting this unique test case.

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 2]
```diff
@@ -36,6 +36,32 @@ public void elementsAreUnique() throws Exception {
...
+        assertNoNulls("hello", "world", ":)");
+        assertNoNulls(1);
+        assertNoNulls("hello");
+        assertNoNulls(1, 2, 3);
+
+        // At least one null object
+        assertAnyNull(null, null);
+        assertAnyNull(null, "value");
+        assertAnyNull(1, 2, 3, null);
+        assertAnyNull("hello", null, "world", null);
```
Comment (AceCentury) [inline test/java/seedu/addressbook/common/UtilsTest.java:54]: It's good that you split your test cases into separate categories and also tested different data types. You might want to consider testing multiple data types together, e.g. assertAnyNull(1,"aString")

**Review by AceCentury — COMMENTED** [review]
Good job for attempting the LO! In future, try to complete them within the same week, preferably before tutorial so that you may maximise your learning. 

If you encounter any difficulties, do consult your teammates for help or contact me :)

Please close this PR when you've read the comments.

**Comment by adeeldb** [comment]
Thank you for the feedback. Duly noted. Closing the commit.

**Pull request closed without merging**